Packages/ClientRuntime/Sources/Networking/Http/Middlewares/Providers/QueryItemProvider.swift

```diff
@@ -6,5 +6,5 @@
 //
 
 public protocol QueryItemProvider {
-    var queryItems: [URLQueryItem] { get }
+    func queryItems() throws -> [URLQueryItem]
```

Comment:
this is a breaking change. Also you can use throwing property

```suggestion
    var queryItems: [URLQueryItem] { get throws }
```

Reply:
Ahh TIL about `get throws` will update. 
This will still be a breaking change though

Reply:
for sure it will be, you may want to update commit message with `!` bang